packages/server/src/fhir/operations/asyncjobcancel.ts

```diff
@@ -24,8 +22,11 @@ export const operation: OperationDefinition = {
 /**
  * Handles HTTP requests for the AsyncJob $cancel operation.
  * Sets the status of the `AsyncJob` to `cancelled`.
+ *
+ * @param req - A request to cancel a particular async job.
+ * @returns A FhirResponse.
  */
-export const asyncJobCancelHandler = asyncWrap(async (req: Request, res: Response) => {
+export async function asyncJobCancelHandler(req: FhirRequest): Promise<FhirResponse> {
```

Comment:
Converted previous handler to proper FHIRRouter handler rather than bare express handler